packages/preset-mini/src/index.ts

```diff
@@ -11,7 +11,7 @@ export { parseColor } from './utils'
 
 export type { ThemeAnimation, Theme }
 
-interface DarkSelector {
+type DarkModeSelectors = ['class', {
```

Comment:
I think a simple object is enough, the array feels redundant to me (plus they are not always `class` anyway)

Reply:
Same as I thought, not sure why `tailwindicss` needs to set a `class`